cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -106,10 +106,11 @@
   @Path("/**/metadata")
   public void getMetadata(HttpRequest request, HttpResponder responder,
                           @QueryParam("scope") String scope, @QueryParam("type") String type,
-                          @QueryParam("showCustom") boolean showCustom)
+                          @QueryParam("showCustom") boolean showCustom,
+                          @DefaultValue("true") @QueryParam("runAggregation") boolean runAggregation)
```

Comment:
it seems a bit odd to me to have 'true' and 'false' for a noun. What about calling it a verb like 'aggregateRun'? Feels more consistent with the 'showCustom' param